src/appengine/handlers/cron/oss_fuzz_setup.py

```diff
@@ -521,6 +524,12 @@ def sync_cf_job(project, info, corpus_bucket, quarantine_bucket, logs_bucket,
     if selective_unpack:
       job.environment_string += 'UNPACK_ALL_FUZZ_TARGETS_AND_FILES = False\n'
 
+    if 'dataflow' in info.get('fuzzing_engines'):
+      dataflow_build_bucket_path = get_build_bucket_path(
+          project, 'dataflow', 'dataflow', template.architecture),
```

Comment:
'dataflow' repeated twice ? dataflow as fuzzing engine makes sense, but then why is it called a memory tool too, shouldn't template.memory_tool be used ?

Reply:
`dataflow` is both a new engine and a new sanitizer. I agree it's kinda annoying, but don't see a better way than specifying it twice.

`template.memory_tool` cannot be used, as we don't have a template for DFsan (we don't create a new job for it), and here `template.memory_tool` points to the sanitizer which fuzzing binaries are built with (e.g. ASan). DataFlow binaries are built with dataflow sanitizer.

"use named arguments" -- done, thanks! Also added a comment to clarify that.